### DISCUSSION TITLE
Add a "bring back edge" option

### DISCUSSION THREAD

**Discussion by anantav51** [ChrisTitusTech/winutil#382] [General]
As of writing this, there's no way for me to get edge back using winutil. I'd like to get it back since I wanna replace my default browser with it.

**Comment by aniketgm** [comment]
I'm afraid, it's gone forever. Chris explains it in [this](https://www.youtube.com/watch?v=vXyMScSbhk4&t=381s) video

**Comment by anonyao** [comment]
Nooo do more delete and break install 🤣
